paasta_tools/cli/cmds/local_run.py

```diff
@@ -79,6 +79,28 @@ class AWSSessionCreds(TypedDict):
     AWS_SECURITY_TOKEN: str
 
 
+def execute_in_container(docker_client, container_id, cmd, timeout):
```

Comment:
i assume this is a move, but is it me or is `timeout` unused?

Reply:
if we clean up the unused param, we can probably also
type this 
```suggestion
def execute_in_container(docker_client: docker.Client, container_id: str, cmd: str) -> Tuple[str, int]:
```